packages/flutter/test/material/outline_button_test.dart

```diff
@@ -1043,6 +1043,93 @@ void main() {
     _checkPhysicalLayer(buttonElement, fillColor.withAlpha(0x00));
   });
 
+  testWidgets('OutlineButton respects the provided materialTapTargetSize', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: Material(
+          child: Center(
+            child: OutlineButton(
+              materialTapTargetSize: MaterialTapTargetSize.padded,
+              child: const SizedBox(width: 50.0, height: 8.0),
+              onPressed: () { },
```

Comment:
nit: we conventionally have the `child` widget be the trailing named parameter in Flutter.
```suggestion
              onPressed: () {},
              child: const SizedBox(width: 50.0, height: 8.0),
```